hpx/runtime/threads/policies/local_priority_queue_scheduler.hpp

```diff
@@ -128,7 +128,7 @@ namespace hpx { namespace threads { namespace policies {
           , affinity_data_(init.affinity_data_)
           , num_queues_(init.num_queues_)
           , num_high_priority_queues_(init.num_high_priority_queues_)
-          , low_priority_queue_(0, thread_queue_init_)
+          , low_priority_queue_(init.num_queues_ - 1, thread_queue_init_)
```

Comment:
Is this just for consistency with `wait_or_add_new` only taking low priority tasks on the last thread? AFAICT we don't actually use the queue index in `thread_queue` or the backends, so we might want to consider removing that argument unless there's a use case for it.

Reply:
Done. I have removed the index from the thread queue.